server/feeders/reservationsgenerator.js

```diff
@@ -56,29 +55,38 @@ var feedOrionReservations = function() {
   };
 
   Object.keys(restaurantsData).forEach(function(element, pos) {
-    // Call orion to append the entity
-    var restaurantName = restaurantsData[pos].id + '-' + shortid.generate();
+
+    var reservationId = restaurantsData[pos].id + '-' + shortid.generate();
 
     var reservations = ['Cancelled', 'Confirmed', 'Hold', 'Pending'];
 
     var attr = {
       'type': 'FoodEstablishmentReservation',
-      'id': restaurantName,
-      'reservationStatus': utils.randomElement(reservations),
-      'underName': {},
-      'reservationFor': {},
-      'startTime': utils.getRandomDate().getTime(),
-      'partySize': utils.randomIntInc(1, 20)
+      'id': utils.asciiEncode(reservationId),
+      'reservationStatus': {
+        'value': utils.randomElement(reservations)
+      },
+      'underName': {
+        'type': 'Person',
+        'value': 'user' + utils.randomIntInc(1, 10)
+      },
+      'reservationFor': {
+        'type': 'FoodEstablishment',
+        'value': utils.fixedEncodeURIComponent(restaurantsData[pos].name)
```

Comment:
Yes, it does not accept certain characters. Because of that, for example, white spaces and some other characters are replaced by underscore, and with this conversion we cannot generate the original name again. Using the name allow us to use `name` or `id` just applying the conversion as needed.